### PR TITLE
Features/consumer bag screen

### PR DESCRIPTION
**Pull request by OngJiaYuan**
Thank you for your contribution to the repo. 
Before submitting this PR, please make sure:

- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

You may delete the sections as necessary.

### Context
___

Fill this up with some context that will be useful to the reviewer that is looking at your PR.

image
This is the screen that shows when u press the cart there is a back button which can be routed to the previous page
image
There is a dialog box which triggers when item count is at one and the user press the minus sign 
pressing remove will trigger a reload of page which will remove the item like this

image

checking and unchecking the box
adding or subtracting number of item will reflect on the total price and item shown on checkout box

image

you can select all or unselect all by pressing the checkbox with the brand name
image

and the total price and item will be reflected accurately with the changes based on checkbox,plus and minus
### Changes
___

Fill this up with the changes that are included in this PR.

Example:
Added a consumer bag screen(cart screen)
where consumer can look at the products they added and they can also checkout the product 

### Shoutouts
___
Thanks Joseph

### CODE REVIEW

File: lib/screens/consumer_bag/consumer_bag_screen.dart [hunk 1]
```diff
@@ -0,0 +1,424 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:easy_localization/easy_localization.dart';
+import 'package:vue_app/screens/consumer_bag/consumer_bag_mock_data.dart';
+import 'package:vue_app/util/constants.dart';
+import 'package:vue_app/util/util.dart';
+import 'consumer_bag_mock_data.dart';
+
+class consumer_Bag extends StatefulWidget {
```
Comment (wheel1992) [inline lib/screens/consumer_bag/consumer_bag_screen.dart:9]: Name the screen class name as `ConsumerBagScreen`

File: lib/screens/consumer_bag/consumer_bag_screen.dart [hunk 2]
```diff
@@ -0,0 +1,424 @@
...
+  @override
+  _consumer_BagState createState() => _consumer_BagState();
+}
+
+class _consumer_BagState extends State<consumer_Bag> {
```
Comment (wheel1992) [inline lib/screens/consumer_bag/consumer_bag_screen.dart:14]: State name should be `ConsumerBagScreenState`

File: lib/screens/consumer_bag/consumer_bag_screen.dart [hunk 3]
```diff
@@ -0,0 +1,424 @@
...
+    }
+    return totalPrice + checkboxPrice;
+  }
+
+//  int totalItem() {
+//
+//  }
+
+  @override
+  Widget build(BuildContext context) {
```
Comment (wheel1992) [inline lib/screens/consumer_bag/consumer_bag_screen.dart:43]: The build function here is too huge. Make into smaller chunks of functions to improve readability and maintability

File: lib/screens/consumer_bag/consumer_bag_screen.dart [hunk 4]
```diff
@@ -0,0 +1,424 @@
...
+}
+
+Future<AlertDialog> myDialog(
+    BuildContext context, AddedItemsFromBrand inputItem) {
+  return showDialog(
+    context: context,
+    builder: (BuildContext context) {
+      return CupertinoAlertDialog(
+        key: Key('AlertBox'),
+        title: Text('Remove Item'),
```
Comment (wheel1992) [inline lib/screens/consumer_bag/consumer_bag_screen.dart:394]: Careful of missing internationalization

File: lib/screens/consumer_bag/consumer_bag_screen.dart (hunk 3 above)
Comment (OngJiaYuan) [reply lib/screens/consumer_bag/consumer_bag_screen.dart]: This one i need a little time cause when i put the function under the mock data it doesnt change and i cannot bring in the function into the state as it has error in it

File: test/screens/consumer_order/consumer_order_screen_test.dart [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    return find.byIcon(Icons.add);
+  }
+
+  Finder findMinus() {
+    return find.byIcon(Icons.remove);
+  }
+
+  Finder findCheckoutBox() {
+    return find.byKey(Key(LocaleText.textCheckout.tr()));
+    ;
```
Comment (wheel1992) [inline test/screens/consumer_order/consumer_order_screen_test.dart:32]: Remove unused semi-colon

File: test/screens/consumer_order/consumer_order_screen_test.dart [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+  }
+
+  Finder findDialogBox() {
+    return find.byKey(Key('AlertBox'));
+  }
+
+  ;
```
Comment (wheel1992) [inline test/screens/consumer_order/consumer_order_screen_test.dart:39]: Remove unused semi-colon

**Comment by wheel1992** [comment]
[Consumer - Bag](https://trello.com/c/07DMT4Nt/9-consumer-bag)

**Pull request merged**